pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -65,9 +87,14 @@ public static boolean isLinux() {
      */
     public static boolean isCGroupEnabled() {
         try {
-            return Files.exists(Paths.get(CGROUPS_CPU_USAGE_PATH));
+            if (metrics == null) {
+                return false;
```

Comment:
If `jdk.internal.platform.Container` is unavailable, I think we should fall back to `CGROUPS_CPU_USAGE_PATH`.

Reply:
Thanks, I updated it.